examples/usertests/resources/reactiveTestTemplate.java.qute

```diff
@@ -0,0 +1,87 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.hibernate:hibernate-core:5.4.10.Final
+//DEPS junit:junit:4.12
+//DEPS javax.persistence:javax.persistence-api:2.2
+//DEPS org.hibernate.reactive:hibernate-reactive-core:1.0.0.CR4
+//DEPS org.assertj:assertj-core:3.13.2
+//DEPS io.vertx:vertx-core:4.0.3
+//DEPS io.vertx:vertx-unit:4.0.3
+//DEPS io.vertx:vertx-pg-client:4.0.3
+//FILES CoreReactiveTest.java
+//
+//
+
+import javax.persistence.*;
```

Comment:
Not critical, but let's avoid `*` imports.

Reply:
:+1: